### DISCUSSION TITLE
Domain verification not available

### DISCUSSION THREAD

**Discussion by rgavilan** [community/community#49606] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Domain verification for organizations isn't available. The option has dissapeared from main Settings menu, but I found it inside "Developers Settings" > "Publisher verification".

When I clic on "Domain verification" > "Review", it navigate to https://github.com/organizations/myorg/settings/domains (substitute myorg by an organization name) and I get a 404.

<img width="1258" alt="image" src="https://user-images.githubusercontent.com/33056993/223977418-d6a5021e-910b-4429-8761-3046e75074e7.png">

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @rgavilan! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by hamdygamal** [comment]
my organization encountering same issue when we go to "**Publisher Verification**" and click on **"Review"** below the domain verification its redirecting to **404 Not Found Page**

**Comment by uhKieran** [comment]
Glad it isn't just me who fails to find this setting. Hopefully someone finds where it's moved or GitHub fixes the issue.

**Comment by taltstidl** [comment]
It looks like domain verification for organizations is a feature of Enterprise Cloud and not available to users or organizations on the Free, Pro or Team plans on GitHub. I also think it may have been available to everyone at some point, but if it was, it has been removed. The statement from the official [documentation](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization) is reproduced below.

> **Note:** To verify or approve domains, your organization must use GitHub Enterprise Cloud. For more information about how you can try GitHub Enterprise Cloud for free, see "[Setting up a trial of GitHub Enterprise Cloud](https://docs.github.com/en/enterprise-cloud@latest/get-started/signing-up-for-github/setting-up-a-trial-of-github-enterprise-cloud)."

_Edit:_ Although the documentation referenced above still contains the same notice, it does look like this feature was only temporarily removed as it is available again to organizations on Free, Pro and Team plans as of Mar 15, 2023.

**Reply by hamdygamal** [reply]
i may guess that right but if the publisher verification is no longer available for open source or Free organizations 
so they should to remove this feature from the sidebar , and let upgraded organizations see this options , to avoid confusion

**Reply by codytubbs** [reply]
Couple of questions for the others having similar experiences with the missing menu option under `Security` for domain validation:
 
1. Nobody has seen their verification badge disappear, correct?  (this has not been a part of the issue for me, yet).

2. Has anyone tried swapping  their email address or domain to one that was known to be verified, with any success?

I have around 85 verified domains I quite recently put a lot of effort in regarding all of the required records.  Now I'm concerned if I swap my currently chosen visible email with one I'd rather be showing, that the system might not respond so well, and then also (worst-case scenario) toggle the verified status of the one I'm using with success at the moment.

There seems to be other issues at the moment as well, but not a topic for here -- it's just making me wonder if there's some infra issues they're dealing with.

Most likely going to sit on my hands until we all know more.

cheers!

**Comment by wecreateproject** [comment]
**I've been watching this for 5 days. It was updated last night.**

**Domain verification is now active, you can check 👍🏾**

**Reply by taltstidl** [reply]
I can confirm that this feature is available for organizations again (on the Teams plan, should it matter). I just successfully verified a new domain, and it is now showing the "Verified" badge on the main organization page👍
